src/canjs/routing/route-data/route-data.js

```diff
@@ -700,6 +700,12 @@ export class RouteData extends ObservableObject {
       Boolean,
       booleanParsing,
     ),
+    openAutoSchedulerModal: saveJSONToUrlButAlsoLookAtReport_DataWrapper(
```

Comment:
@justinbmeyer you cool with this? This logic controls whether or not the modal for auto scheduler opens or closes. Normally we would use Context to handle this, but because of how everything bootstraps within a Stache element, that's not possible.